src/rdo.rs

```diff
@@ -355,9 +355,48 @@ fn compute_tx_distortion<T: Pixel>(
   distortion
 }
 
-pub fn compute_rd_cost<T: Pixel>(fi: &FrameInvariants<T>, rate: u32, distortion: u64) -> f64 {
+pub fn compute_rd_cost_biased_by_importance<T: Pixel>(
+  fi: &FrameInvariants<T>, rect: Rect, rate: u32, distortion: u64
+) -> f64 {
+  assert!(rect.x >= 0);
```

Comment:
This function is hot (called multiple times per block) so this needs to be debug_assert!().